$NetBSD: patch-Makefile.am,v 1.3 2020/12/26 05:23:12 mef Exp $

Make sure config.h is not copied at install time, we do not need it.

--- Makefile.am.orig	2019-03-15 04:24:46.000000000 +0000
+++ Makefile.am
@@ -1,6 +1,6 @@
 ACLOCAL_AMFLAGS = -I m4
 
-nobase_pkginclude_HEADERS = config.h
+#nobase_pkginclude_HEADERS = config.h
 
 GEN_DRV =
 PLAT_DRV =
